driver-core/src/main/java/com/datastax/driver/core/AbstractTableMetadata.java

```diff
@@ -240,7 +248,7 @@ protected StringBuilder appendOptions(StringBuilder sb, boolean formatted) {
         sb.append(" WITH ");
         if (options.isCompactStorage())
             and(sb.append("COMPACT STORAGE"), formatted);
-        if (!Iterables.all(clusteringOrder, isAscending))
+        if (!clusteringOrder.isEmpty())
```

Comment:
This change isn't necessary, but I noticed this is what the python driver does, and we are already pretty aggressive about including options which the user wouldn't (usually) specify themselves.